dictionary/homework_dictionary.py

```diff
@@ -0,0 +1,55 @@
+"""Solution task 1"""
+check_name = input("Please, enter your name:  ").title()
+roles_dict = {
+    'admin': ['George', 'Ruslan', 'Valeriy', 'Irina'],
+    'maintainer': ['Julia', 'Alex'],
+    'manager': ['Piter', 'Max', 'Sophia'],
+    'developer': ['Natali', 'Nikita', 'Diana']
+}
+list_system_names = []
+system_role = ''
+
+for roles, list_names in roles_dict.items():
+    for names in list_names:
+        list_system_names.append(names)
+        if check_name in names:
+            system_role = roles
+        else:
+            pass
+"""I think for correcting end current loops should be block else,
+but I don`t know what to write in this block else.
+Is it correct if i write else: pass?"""
+
+if check_name in list_system_names:
+    print(f"Hello {system_role}")
+else:
+    print("Hello guest")
```

Comment:
Can be better
![image](https://github.com/Pticatos86/test_repos_python/assets/51053483/c41fda02-7a50-40b8-bdf6-6e34ec909dd6)

Reply:
fixed block loop